app/assets/stylesheets/new_board/_story.scss

```
@@ -341,11 +341,21 @@
         &__controls {
           @extend .btn-group;
 
+          height: 22px;
+          display: block;
+
           input {
             @extend .btn;
             @extend .btn-default;
             @extend .btn-xs;
           }
+          
+          .infoToolTip {
+            background-color: #fff;
```

There is a file which contains all variables for colors, you can use $white here.

```suggestion
            background-color: $white;
```

---

do you think we really need that background? I would remove it a leave just the icon with transparent background.